mobile/library/cc/engine_builder.cc

```diff
@@ -856,6 +875,7 @@ std::unique_ptr<envoy::config::bootstrap::v3::Bootstrap> EngineBuilder::generate
   list->add_patterns()->set_exact("cluster.stats.http2.keepalive_timeout");
   list->add_patterns()->set_prefix("http.hcm.downstream_rq_");
   list->add_patterns()->set_prefix("http.hcm.decompressor.");
+  list->add_patterns()->set_prefix("cluster_manager.");
```

Comment:
one thing Alyssa pointed out, `cluster_manager.` prefix may have a lot of stats.  We probably want to limit this to the ones we care about.  Might be good to take stock of the metrics and which ones we'll care about for mobile CDS and then only allow-list those patterns.  Also, we probably only want to add this prefix *if* enable_cds_ is true.